addon/styles/components/text-field.scss

```diff
@@ -1,76 +1,122 @@
 .cs-component-text-field {
-  &__input {
-    font-family: inherit;
-    font-size: 18px;
-    font-weight: 300;
-    border-radius: 2px;
-    margin: 0;
-    padding-left: 15px;
-    padding-top: 20px;
-    width: 100%;
-    background: $lighter-gray;
-    transition: padding-top 0.2s ease, margin-top 0.2s ease;
-    overflow-x: hidden; /* Hack to make "rows" attribute apply in Firefox. */
-    height: 60px;
-    border: 1px solid $border-gray;
-    box-sizing: border-box;
-  }
+  // local variables
+  --cs-component-text-field-border-radius: var(--cs-input-border-radius);
+  --cs-component-text-field-label: var(--cs-input-label);
+  --cs-component-text-field-validation: var(--cs-input-valid);
+  --cs-component-text-field-validation-icon: url('/@cardstack/ui-components/images/check-icon.svg');
+  --cs-component-text-field-validation-icon-size: 1em;
 
-  &__input + label {
-    display: block;
-    position: relative;
-    white-space: nowrap;
-    padding: 0;
-    margin: -10px 15px 0 15px;
-    height: 0px;
-    border-bottom: 1px solid $border-gray;
-  }
+  position: relative;
+  padding-bottom: 3em;
```

Comment:
this padding is quite high, we should let the component's container determine the spacing.

<img width="529" alt="Screen Shot 2019-05-13 at 11 27 03 AM" src="https://user-images.githubusercontent.com/176297/57633843-2e98d400-7572-11e9-8120-b21c3119bd2e.png">

Reply:
The padding is there for the confirmation/validation messages. I'll drop it to 2em. Any lower and the element below will overlap with the messages.